misc/strip-bouncycastle/pom.xml

```diff
@@ -33,12 +33,12 @@
         <dependency>
             <groupId>org.bouncycastle</groupId>
             <artifactId>bcprov-jdk15on</artifactId>
-            <version>1.59</version>
+            <version>${bouncycastle.version}</version>
```

Comment:
Shall we move these dependencies to the parent pom and refer those from here without the version??